content/html/concepts/tags/terms/video/video.md

```diff
@@ -52,6 +52,6 @@ The `<video>` tag may include `<source>` tags and fallback text. It may also use
 </html>
 ```
 
-<video>
-  <source type="video/mp4" src="https://github.com/Codecademy/docs/blob/5be38e91f804a5d035bc33df4fe29c5c5243a117/media/html-video-tag-example.mp4?raw=true" />
+<video src="https://raw.githubusercontent.com/Codecademy/docs/main/media/html-video-tag-example.mp4?raw=true">
+  <h3>This video is not supported.</h3>
 </video>
```

Comment:
```suggestion
<video controls>
  <source type="video/mp4" src="https://raw.githubusercontent.com/Codecademy/docs/main/media/html-video-tag-example.mp4" />
</video>
```

Reply:
Actually, let's stick with using the inner `<source>` tag but with `controls` added to outer `<video>` tag.